python-avd/pyavd/_anta/logs.py

```diff
@@ -83,6 +83,7 @@ class LogMessage(Enum):
 
     # Interface state messages
     INTERFACE_SHUTDOWN = "{interface} skipped - Interface is shutdown"
+    PEER_INTERFACE_SHUTDOWN = "{interface} skipped - Peer interface is shutdown"
```

Comment:
```suggestion
    PEER_INTERFACE_SHUTDOWN = "{interface} skipped - peer {peer} interface {peer_interface} is shutdown"
```
Also, move this with the other "Peer-related messages" above. Thanks!

Reply:
done